libvast/vast/format/reader.hpp

```diff
@@ -110,7 +110,7 @@ class reader {
 
   caf::atom_value table_slice_type_;
   std::chrono::steady_clock::duration read_timeout_
-    = vast::defaults::import::read_timeout;
+    = std::chrono::milliseconds{20};
```

Comment:
Let's agree on a sensible value here before touching `defaults.hpp`.

Reply:
I'm fine with the value, but this should still be changed to use the value set in `defaults.hpp`.